webpack.karma.js

```diff
@@ -0,0 +1,43 @@
+const postcssImport = require('postcss-import');
+
+module.exports = {
+  devtool: 'inline-source-map',
+  module: {
+    preLoaders: [
+      {
+        test: /.*[^.spec].(js|jsx)$/,
+        exclude: /node_modules/,
+        loader: 'babel-istanbul',
+        query: {
+          cacheDirectory: true,
+        }
+      },{
+        test: /.spec.js$/,
+        exclude: /node_modules/,
+        loaders: ['babel'],
+      }
+    ],
+    loaders: [
+      {
+        test: /\.css$/,
+        exclude: /node_modules/,
+        loaders: ['style', 'css?modules&importLoaders=1', 'postcss'],
+      }, {
+        test: /\.json$/,
+        loader: 'json',
+      },
+    ]
+  },
+  postcss: (webpack) => {
+    return [
+      postcssImport({
+        addDependencyTo: webpack,
```

Comment:
There are some common element, between this file and webpack.config.js, maybe we could think about having a base config that both these files require from so that we don't have to keep the files in sync?